weaviate/templates/querierService.yaml

```diff
@@ -0,0 +1,36 @@
+{{ if .Values.experimental.querier.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.experimental.querier.service.name }}
+  labels:
+    app.kubernetes.io/name: weaviate
```

Comment:
can we agree on `weaviate-querier` naming? to be used throughout all of those files?